edx_django_utils/plugins/docs/decisions/0003-plugin-contexts.rst

```diff
@@ -0,0 +1,88 @@
+Plugin Contexts
+---------------
+
+Status
+======
+Draft
```

Comment:
How old is this ADR?  I presume it is accepted by now, given it is in use.

Reply:
I'm changing this to accepted